notebooks/basics/quantum-circuits.ipynb

```diff
@@ -807,7 +812,7 @@
     "\n",
     "Geometrically, we can think about orthogonal vectors as vectors at right angles to each other.\n",
     "\n",
-    "A set of vectors $\\{ \\vert \\psi_1\\rangle,\\ldots,\\vert\\psi_m\\rangle\\}$ is called an *orthogonal set* if it is the case that every vector in the set is orthogonal to every other vector in the set.\n",
+    "A set of vectors $\\{ \\vert \\psi_1\\rangle,\\ldots,\\vert\\psi_m\\rangle\\}$ is called an *orthogonal set* if every vector in the set is orthogonal to every other vector in the set.\n",
     "That is, this set is orthogonal if\n",
     "\n",
     "$$\n",
```

Comment:
Not sure how to add a comment below... but I made a mistake with the tag: `\tag{2.3}` should be `\tag{3}`